app/controllers/application_controller.rb

```diff
@@ -9,7 +9,7 @@ class ApplicationController < ActionController::Base
     if: -> { ENV.key?("BASIC_AUTH_USERNAME") },
   )
 
-  helper_method :admin_signed_in?, :current_claim
+  helper_method :admin_signed_in?, :current_claim, :service_operator_signed_in?
```

Comment:
This change should happen in the base admin controller where the method is defined.

Reply:
Oh, good shout. I thought I'd done that